src/utils/fetchClient.ts

```diff
@@ -1,32 +1,28 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the 'eslint-disable' directive and ensure that the 'any' type is not used explicitly. If needed, use a more specific type or create an interface for the data.